lib/src/shared/status.dart

```diff
@@ -155,62 +155,73 @@ class GrpcError implements Exception {
   final int code;
   final String? message;
   final Object? rawResponse;
+  final Map<String, String>? trailers;
   final List<GeneratedMessage>? details;
 
   /// Custom error code.
-  GrpcError.custom(this.code, [this.message, this.details, this.rawResponse]);
```

Comment:
This is a breaking change in a public constructor. Could we avoid it?

Reply:
Agree, I'll refactor.